src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYouWhenFromMyCollection.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Flex, Text } from "@artsy/palette"
+import { useSystemContext } from "System"
+import { RouterLink } from "System/Router/RouterLink"
+
+export const ThankYouWhenFromMyCollection: React.FC = () => {
+  const { isLoggedIn } = useSystemContext()
+
+  return (
+    <>
+      <Text variant="xl" mt={4}>
+        Your artwork has been submitted
+      </Text>
+      <Text variant="md" color="black60" mt={1} mb={[4, 6]}>
+        We will email you within 1-3 days to confirm if your artwork has been
+        accepted or not.
+      </Text>
+
+      <Flex
+        flexDirection={["column", "row"]}
+        alignItems={["stretch", "center"]}
+      >
+        {isLoggedIn ? (
+          <RouterLink to="/my-collection">
+            <Button
+              width={["100%", "auto"]}
+              data-test-id="back-to-my-collection"
+              size="large"
+              variant="primaryBlack"
+            >
+              Back to My Collection
+            </Button>
+          </RouterLink>
+        ) : (
+          <RouterLink to="/">
+            <Button
+              width={["100%", "auto"]}
+              data-test-id="back-to-artsy-homepage"
+              size="large"
+              variant="primaryBlack"
+            >
+              Back to Artsy Homepage
+            </Button>
+          </RouterLink>
+        )}
```

Comment:
**Suggestion:** maybe we could remove the non-logged-in flow since we won't have my collection without the user being logged in

Reply:
I saw you added these changes in [this PR](https://github.com/artsy/force/pull/10906), so I left it as it is. Should I revert it?

Reply:
Yeah, I only added it to allow non-logged-in users to submit through SWA flow, but now I don't see it relevant anymore :)

Reply:
Ah, alright, I'll remove it